### PR TITLE
Bump version to 1.0.7

### PR DESCRIPTION
**Pull request by dev-nis**
- Fix Icon size (thanks to https://github.com/carmenfdezb )
- Add Spanish translation (thanks to https://github.com/carmenfdezb )
- Add German translation
- Fix the build for aarch64 on modern SailfishOS-versions:
- Changed call to systemd to new path
- Fixed inclusion of systemd-service in RPM packaging

### CODE REVIEW

File: jolla-settings-screenmonitor.pro.user.0a580b9
Comment (g7) [inline jolla-settings-screenmonitor.pro.user.0a580b9:1]: Is this needed? Looks some qtcreator user specific settings (I think they can be gitignored)

File: service/ScreenMonitor
Comment (g7) [inline service/ScreenMonitor:1]: Build artifact - can be removed and gitignored

**Review by g7 — CHANGES_REQUESTED** [review]
Thanks for resurrecting this! :100: 

Some minor comments for some unneeded artifacts that can be gitignored, otherwise looks good to me

File: jolla-settings-screenmonitor.pro.user.0a580b9
Comment (dev-nis) [reply jolla-settings-screenmonitor.pro.user.0a580b9:1]: I don't think it's needed. I removed it and added it to .gitignore

File: service/ScreenMonitor
Comment (dev-nis) [reply service/ScreenMonitor:1]: Done!

**Comment by g7** [comment]
Thank you very much! Sorry for merging this late

**Review by g7 — APPROVED** [review]

**Comment by dev-nis** [comment]
> Thank you very much! Sorry for merging this late

No problem!
Should I remove my forked version from OpenRepos then?

**Pull request merged**